### PR TITLE
DEV-37 Migrated architecture to CRA

### PR DESCRIPTION
**Pull request by KovDimaY**
For more info check #85

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,54 +1,40 @@
 {
-  "name": "highcharts-test",
-  "version": "0.0.0",
+  "name": "temp",
```
Comment (TetianaKlymchuk) [inline package.json]: Maybe is better to rename it 🤔

File: src/index.js [hunk 2]
```diff
@@ -4,9 +4,16 @@ import { Router, browserHistory } from 'react-router';
...
 
 ReactDOM.render(
-  <Router history={browserHistory} routes={routes} />,
-  document.querySelector('#react-app')
+  <React.StrictMode>
+    <Router history={browserHistory} routes={routes} />
+  </React.StrictMode>,
+  document.getElementById('root')
 );
+
```
Comment (TetianaKlymchuk) [inline src/index.js]: We need prettier asap

**Review by TetianaKlymchuk — CHANGES_REQUESTED** [review]
Good first step! 👍

File: src/index.js (hunk 2 above)
Comment (KovDimaY) [reply src/index.js]: Fixed

File: package.json (hunk 1 above)
Comment (KovDimaY) [reply package.json]: Fixed

**Review by TetianaKlymchuk — APPROVED** [review]

**Pull request merged**